src/richtext/richtextbuffer.cpp

```diff
@@ -6529,14 +6530,16 @@ wxRichTextLine* wxRichTextParagraph::AllocateLine(int pos)
 {
     if (pos < (int) m_cachedLines.GetCount())
     {
-        wxRichTextLine* line = m_cachedLines.Item(pos)->GetData();
+        assert(m_cachedLinesVect.size() == m_cachedLines.GetCount());
```

Comment:
We shouldn't use `assert()` in wx code, please use either `wxASSERT()` or `wxCHECK()` if you really need to.